weavelet.go

```diff
@@ -108,7 +108,7 @@ func newWeavelet(ctx context.Context, componentInfos []*codegen.Registration) (*
 	}
 	w.info = wletInfo
 
-	exporter, err := env.CreateTraceExporter()
+	exporter := env.CreateTraceExporter()
 	if err != nil {
```

Comment:
This `if` is stale now?

Reply:
Yes, good catch! I moved the call into `sdktrace.WithBatcher()` since that was the only place the `exporter` variable was used.